benchmark/fft/fft_benchmark.cc

```diff
@@ -82,94 +82,94 @@ void Run(const FFTConfig& config) {
       config.exponents(),
       [](uint32_t exponent) { return base::NumberToString(exponent); }));
 
+  reporter.AddVendor(Vendor::Tachyon());
+  for (Vendor vendor : config.vendors()) {
+    reporter.AddVendor(vendor);
+  }
```

Comment:
If we see the function in`FFTRunner::Run()` and `FFTRunner::RunExternal()`, it seems not needed.

Reply:
I removed from `Run()` and `RunExternal()` because we call it multiple times for the same vendor.